cli/src/commands/start.rs

```diff
@@ -45,18 +45,25 @@ pub struct Start {
     #[clap(default_value = "3", long = "network")]
     pub network: u16,
 
-    /// Specify this node as a beacon, with the account private key as an argument
+    /// Specify this node as a beacon
     #[clap(long = "beacon")]
-    pub beacon: Option<String>,
-    /// Specify this node as a validator, with the account private key as an argument
+    pub beacon: bool,
+    /// Specify this node as a validator
     #[clap(long = "validator")]
-    pub validator: Option<String>,
-    /// Specify this node as a prover, with the account private key as an argument
+    pub validator: bool,
+    /// Specify this node as a prover
     #[clap(long = "prover")]
-    pub prover: Option<String>,
-    /// Specify this node as a client, with an optional account private key as an argument
+    pub prover: bool,
+    /// Specify this node as a client
     #[clap(long = "client")]
-    pub client: Option<String>,
+    pub client: bool,
+
+    /// Specify the node's account private key.
+    #[clap(long = "private-key")]
+    pub private_key: Option<String>,
+    /// Specify the path to the node's account private key.
```

Comment:
We may want to provide more clarity to the user that this is the `file path` and perhaps the `absolute file path` (or relative if that is the case)